scripts/fast_build.sh

```
@@ -0,0 +1,120 @@
+echo -e "${YELLOW}Building in $PROFILE mode...${NC}"
+
+# Check if librocksdb-sys is already compiled
+TARGET_DIR="target/$PROFILE"
+if ls $TARGET_DIR/deps/*rocksdb*.rlib 1> /dev/null 2>&1; then
+    echo -e "${GREEN}✓ librocksdb-sys already compiled, using cache${NC}"
+else
+    echo -e "${YELLOW}⚠ librocksdb-sys not found in cache, will compile...${NC}"
+fi
+
+# Set environment variables for faster compilation
+export CARGO_BUILD_JOBS=$(nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 4)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)